### PR TITLE
Adds the option to go without a working tongue at roundstart!

### PR DESCRIPTION
**Pull request by Thesingleworstcoder**
## About The Pull Request

Simply adds the option to have your tongue replaced with a stump. Why it's this way is completely up to you, I'm just giving you the option to do it.

## Why It's Good For The Game

Because it's a cool character concept that isn't currently possible in the game. Having a character who can't talk will always be interesting!

## Changelog
:cl:
add: Added tongue stump to the augmentation menu.
:cl:

### CODE REVIEW

**Comment by Thesingleworstcoder** [comment]
No conflicts to speak of!

**Comment by SpaceVampire** [comment]
Well, we already have a mute quirk.
So I am not sure if it creates a "cool character concept that isn't currently possible in the game" as you could just take that.
With your tongue being the flavor reason for it. Because you set this to give points.
You now just get double points for going mute, because there is no reason to not take both of em.

**Comment by Thesingleworstcoder** [comment]
That is a good point, want me to make it so that it's zero points instead?

**Comment by Thesingleworstcoder** [comment]
> 
> 
> Well, we already have a mute quirk.
> So I am not sure if it creates a "cool character concept that isn't currently possible in the game" as you could just take that.
> With your tongue being the flavor reason for it. Because you set this to give points.
> You now just get double points for going mute, because there is no reason to not take both of em.

Changed it to be zero points, now it's truly a flavour choice!

**Comment by Thesingleworstcoder** [comment]
Pretty sure the test-fail was unrelated to what I've done, it had something to do with the Makarov code?

File: code/modules/surgery/organs/tongue.dm [hunk 1]
```diff
@@ -485,3 +485,20 @@
 		signer.visible_message(span_notice("[signer] raises [signer.p_their()] eyebrows."))
 	else if(question_found)
 		signer.visible_message(span_notice("[signer] lowers [signer.p_their()] eyebrows."))
+
+/obj/item/organ/tongue/stump
```
Comment (ZephyrTFA) [inline code/modules/surgery/organs/tongue.dm]: You should probably move this to a modular file. Some where in the customization module likely

Comment (Thesingleworstcoder) [reply code/modules/surgery/organs/tongue.dm]: Alright, I'll do that quickly!

**Comment by Thesingleworstcoder** [comment]
Done!

File: code/modules/surgery/organs/tongue.dm (hunk 1 above)
Comment (Thesingleworstcoder) [reply code/modules/surgery/organs/tongue.dm]: Done that now!

File: modular_skyrat/modules/customization/modules/client/augment/organs.dm [hunk 2]
```diff
@@ -89,3 +89,26 @@
...
+/obj/item/organ/tongue/stump
+	name = "Tongue stump"
+	desc = "What remains of a tongue."
+	icon_state = "tonguenormal"
+	sense_of_taste = FALSE
+	attack_verb_continuous = list("drools", "spits", "gurgles")
+	attack_verb_simple = list("drool", "spit", "gurgle")
+	modifies_speech = FALSE
+
+/obj/item/organ/tongue/stump/Insert(mob/living/carbon/tongue_owner, special = 0)
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/augment/organs.dm]:
```suggestion
/obj/item/organ/tongue/stump/Insert(mob/living/carbon/tongue_owner)
```

File: modular_skyrat/modules/customization/modules/client/augment/organs.dm [hunk 3]
```diff
@@ -89,3 +89,26 @@
...
+	. = ..()
+	ADD_TRAIT(tongue_owner, TRAIT_MUTE, ORGAN_TRAIT)
+
+/obj/item/organ/tongue/stump/Remove(mob/living/carbon/tongue_owner, special = 0)
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/augment/organs.dm:108]:
```suggestion
/obj/item/organ/tongue/stump/Remove(mob/living/carbon/tongue_owner)
```

File: modular_skyrat/modules/customization/modules/client/augment/organs.dm [hunk 4]
```diff
@@ -89,3 +89,26 @@
...
+	..()
```
Comment (ZephyrTFA) [inline modular_skyrat/modules/customization/modules/client/augment/organs.dm:109]:
```suggestion
	. = ..()
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Comment by hugmeorelse** [comment]
I fail to see a significant difference between this and mute. What value is added from this? It's not like you can't take mute and claim your tongue was cut out.

**Comment by Thesingleworstcoder** [comment]
>I fail to see a significant difference between this and mute. What value is added from this? It's not like you can't take mute and claim your tongue was cut out.

You're looking at it the wrong way, because it makes no sense for us to have characters "claiming" things about themselves when we can just make it a feature to add to the game. It's a tangible, if minor, difference that could make or break someone's immersion. Plus, it takes almost no space in the code, so we're not losing anything by gaining this.

**Pull request closed without merging**